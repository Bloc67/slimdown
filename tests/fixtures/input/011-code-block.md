```
function foo () {
	print ('*Foo* _bar_');
}
```