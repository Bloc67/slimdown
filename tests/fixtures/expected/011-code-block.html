<pre><code>function foo () {<br>	print (&#039;Foo&#039;);<br>}</code></pre>